packages/jira-adapter/src/filters/queue.ts

```diff
@@ -0,0 +1,159 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import { Change, InstanceElement, getChangeData, isAdditionChange, isInstanceChange, isRemovalChange } from '@salto-io/adapter-api'
+import { createSchemeGuard, getParent, resolveChangeElement } from '@salto-io/adapter-utils'
+import _ from 'lodash'
+import { collections } from '@salto-io/lowerdash'
+import { elements as elementUtils } from '@salto-io/adapter-components'
+import Joi from 'joi'
+import { deployChanges } from '../deployment/standard_deployment'
+import { FilterCreator } from '../filter'
+import { QUEUE_TYPE } from '../constants'
+import { getLookUpName } from '../reference_mapping'
+import JiraClient from '../client/client'
+
+const { replaceInstanceTypeForDeploy } = elementUtils.ducktype
+const { awu } = collections.asynciterable
+type queueGetRsponse = {
+  values: {
+    id: string
+    name: string
+  }[]
+}
+const QUEUE_RESOPNSE_SCHEME = Joi.object({
+  values: Joi.array().items(Joi.object({
+    id: Joi.string().required(),
+    name: Joi.string().required(),
+  }).unknown(true)).required(),
+}).unknown(true).required()
+
+const isQueueResponse = createSchemeGuard<queueGetRsponse>(QUEUE_RESOPNSE_SCHEME)
+
+const isDefaultNameAddition = async (
+  change: Change<InstanceElement>,
+  client: JiraClient,
+):Promise<boolean> => {
+  const instance = getChangeData(change)
+  const parent = getParent(instance)
+  const { serviceDeskId } = parent.value
+  if (serviceDeskId === undefined) {
```

Comment:
maybe  - 
```suggestion
  if (!_.isString(serviceDeskId)) {
```

this way we verify it's a string and not something else?